src/main/java/org/apache/commons/compress/archivers/zip/ZipSplitReadOnlySeekableByteChannel.java

```diff
@@ -47,7 +50,7 @@
 public class ZipSplitReadOnlySeekableByteChannel extends MultiReadOnlySeekableByteChannel {
     private static final int ZIP_SPLIT_SIGNATURE_LENGTH = 4;
     private final ByteBuffer zipSplitSignatureByteBuffer =
-        ByteBuffer.allocate(ZIP_SPLIT_SIGNATURE_LENGTH);
+            ByteBuffer.allocate(ZIP_SPLIT_SIGNATURE_LENGTH);
```

Comment:
Please do not change the formatting, it makes reviewing harder and take more time.

Reply:
My mistake, reverted!